source/content/drupal-9-cache.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Drupal 8 Performance and Global CDN Caching Settings
-description: Optimize Drupal 8 and Glocal CDN caching to maximize your Pantheon site's performance.
+title: Drupal 9 Performance and Global CDN Caching Settings
```

Comment:
Will the old content/drupal-8-cache path redirect to drupal-9-cache, or 404?

Reply:
It will redirect.